train.py

```diff
@@ -146,8 +147,8 @@
             args.brands_data_file(),
             code_col=0,
             description_col=1,
-            cleaning_pipeline=basic_pipeline,
-            multiplier=3,
+            cleaning_pipeline=brands_pipeline,
+            multiplier=5,
```

Comment:
Some of the brand references weren't appearing in results due to collisions with things coming in for the tradeset data so boosting brands a bit, here to better reflect brand references